.github/PULL_REQUEST_TEMPLATE

```diff
@@ -1,3 +1,12 @@
+## Before you submit a Pull Request
+The Kubernetes scheduler backend is currently being upstreamed to the main [Apache Spark project](https://github.com/apache/spark).
+We are attempting to re-direct as much new development as possible to the upstream.
+Please consider whether your pull request can be submitted against the Apache Spark project, and submit there if possible.
+
+If you have any questions about whether a PR should be submitted upstream or against this fork,
+please feel free to ask on the Big Data SIG [slack channel](https://kubernetes.slack.com/),
```

Comment:
Let's also add a link to the spark-dev mailing list here, and encourage use of that. Slack is good to keep around for questions about "where should this feature go?".